src/libraries/Microsoft.PowerFx.Interpreter/Functions/StandardErrorHandling.cs

```diff
@@ -362,31 +409,38 @@ private static List<List<T>> Transpose<T>(List<List<T>> columns, int columnSize)
          * As a concrete example, Concatenate(["a", "b"], ["1", "2"]) => ["a1", "b2"]
         */
         public static Func<EvalVisitor, EvalVisitorContext, IRContext, FormulaValue[], ValueTask<FormulaValue>> MultiSingleColumnTable(
-            AsyncFunctionPtr targetFunction,
-            bool transposeEmptyTable)
+            AsyncFunctionPtr targetFunction)
         {
             return async (runner, context, irContext, args) =>
             {
                 var resultRows = new List<DValue<RecordValue>>();
 
-                (var maxSize, var emptyTablePresent) = AnalyzeTableArguments(args);
-                if (maxSize == 0 || (emptyTablePresent && !transposeEmptyTable))
+                (var maxSize, var minSize, var emptyTablePresent, var blankTablePresent, _) = AnalyzeTableArguments(args, irContext);
+
+                // If one arg is blank table and among all other args, return blank.
+                // e.g. Concatenate(Blank(), []), Concatenate(Blank(), "test"), Concatenate(Blank(), ["test"], []) => Blank()
+                if (blankTablePresent)
+                {
+                    return FormulaValue.NewBlank();
+                }
+
+                if (maxSize == 0)
                 {
                     // maxSize == 0 means there are no tables with rows. This can happen when we expect a Table at compile time but we recieve Blank() at runtime,
```

Comment:
>This can happen when we expect a Table at compile time but we recieve Blank() at runtime [](http://example.com/codeflow?start=73&length=88)

Can you update the comment, since this is not correct? It can be something like `This can happen if we receive a Filter expression where no rows were returned`